packages/crypto/tests/fixtures/test-vectors/aes-ctr/decrypt.json

```diff
@@ -0,0 +1,35 @@
+{
+  "description": "AES-CTR decrypt test vectors",
```

Comment:
eventually we can add these to the sdk-development test vectors

Reply:
Yep!  Started using the new format so that its easy to pull them in to `sdk-development`.